src/constants.ts

```diff
@@ -155,6 +155,11 @@ export const AUTHORITY_TOKEN_DATA = TOKEN_AUTHORITY_MASK | 1;
  */
 export const HATHOR_TOKEN_CONFIG = { name: 'Hathor', symbol: 'HTR', uid: '00' };
 
+/**
+ * Hathor token uid
+ */
+export const HATHOR_TOKEN_UID: string = '00';
```

Comment:
Why don't we call it `DEFAULT_TOKEN_UID`?

Reply:
Another idea on top of this is to keep the `HATHOR_TOKEN_UID` as `00`, because that's what the Hathor Network actually uses.

Another constant could be `NATIVE_TOKEN_UID`, and this one could be obtained from the `config.ts` instead of `constants.js`. That way one would not need to recompile the application just to refer to a different network.

Reply:
Renamed to `NATIVE_TOKEN_UID` as discussed.